### PR TITLE
Collection as text

### PR DESCRIPTION
**Pull request by memoyil**
Collection as text

### CODE REVIEW

**Comment by 0crat** [comment]
@memoyil I'm not managing this repo, remove the [webhook](https://github.com/yegor256/cactoos/settings/hooks) or contact me in [Slack](http://www.zerocracy.com) //cc @yegor256

File: src/main/java/org/cactoos/text/CollectionAsText.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+     * The delimeter.
+     */
+    private final CharSequence delimeter;
+
+    /**
+     * Ctor.
+     * @param collection The Collection
+     * @param seperator The seperator
+     */
+    public CollectionAsText(final Collection<? extends String> collection, final
```
Comment (Timmeey) [inline src/main/java/org/cactoos/text/CollectionAsText.java]: String is final, therefore `<? extends String>` is redundant

**Comment by Timmeey** [comment]
Take a look at #80 https://github.com/yegor256/cactoos/pull/80/files#diff-0e5a852a570d9dcfd2b2c143ee8f2d17

```
Collection<String> stringsCollection=......
new JoinedText(
  new TransformedIterable(
    stringsCollection,
    new StringAsTextFunc())
```
will exactly do what you propose.
So if you think that should be a buildIn operation, try to compose this witht eh already present classes like the way i sceteched it above (the PR is not merged yet, but it will be soon)

**Comment by memoyil** [comment]
@Timmeey I think JoinedText and CollectionAsText are different issues and both of them should be inside the API.

**Comment by Timmeey** [comment]
But isn't that exactly what you do?
You put in a Collection of Strings (which can be seen as an iterable, since COllection implements iterable)
And you get out one single long text, join on your charSequence Delimiter.

JoinedText takes a collection/iterable of Texts, 
and puts out a long text, joined on the delimiter.

Sorry, i don't really see the difference here.

Maybe give an example of how you would use them differently?

**Comment by memoyil** [comment]
@Timmeey  Okay, that's correct. I accept your critics. Let's get another comments from other contributors and Yegor also. I dont say that mine's correct and we should apply this PR. I think so as you also think like that.

**Comment by yegor256** [comment]
@memoyil this one seems redundant to me if we have `JoinedText`

**Comment by memoyil** [comment]
@yegor256 Ok i am closing the pull request

File: src/main/java/org/cactoos/text/CollectionAsText.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+     */
+    private final CharSequence delimeter;
+
+    /**
+     * Ctor.
+     * @param collection The Collection
+     * @param seperator The seperator
+     */
+    public CollectionAsText(final Collection<String> collection, final
+        CharSequence seperator) {
```
Comment (englishman) [inline src/main/java/org/cactoos/text/CollectionAsText.java]: Can we put this on prev. line?

File: src/main/java/org/cactoos/text/CollectionAsText.java [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        this.source = collection;
+        this.delimeter = seperator;
+    }
+
+    @Override
+    public String asString() {
+        return this.source.stream()
+            .map(i -> i.toString())
```
Comment (englishman) [inline src/main/java/org/cactoos/text/CollectionAsText.java]: Don't needed at all as `source` defined already as `Collection<String> source`

File: src/test/java/org/cactoos/text/CollectionAsTextTest.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+ * @checkstyle JavadocMethodCheck (500 lines)
+ */
+public final class CollectionAsTextTest {
+
+    @Test
+    public void collectionAsText() {
+        MatcherAssert.assertThat(
+            "Can't contruct a text from collection",
+            new CollectionAsText(
+                new ArrayList<String>(Arrays.asList("Hello", "World")), ","
```
Comment (englishman) [inline src/test/java/org/cactoos/text/CollectionAsTextTest.java]: `new ArrayList<String>(...)` wrapper isn't needed here

**Pull request closed without merging**